image/preseed/preseed_uc20_test.go

```diff
@@ -285,6 +285,11 @@ func (s *preseedSuite) testRunPreseedUC20Happy(c *C, customAppArmorFeaturesDir s
 	c.Assert(err, IsNil)
 	defer r.Close()
 
+	// check directory targetSnapdRoot was deleted
+	_, err = os.Stat(targetSnapdRoot)
```

Comment:
(nitpick) We have osutil.FileExists() and osutil.DirectoryExists() but this is fine too, it's just a test.